lib/typedefs.js

```diff
@@ -173,6 +191,21 @@
  */
 
 /**
+ * The shape of the PR pulled from Bitbucket's Cloud `/repositories/:owner/:repo/pullrequests` API
+ * {@link https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D}
+ *
+ * @typedef BitbucketPullRequest
+ * @property {Number} id - the PR #
+ * @property {String} description - the description of the PR
+ * @property {BitbucketEndpoint} source - the ref of the source of the pr
+ * @property {BitbucketEndpoint} destination - the ref of the source of the pr
+ * @property {BitbucketCommit} source.commit - the ref of the source commit of the pr
+ * @property {BitbucketCommit} destination.commit - the ref of the destination commit of the pr
```

Comment:
It seems like these two lines, and the `links.self` line below are rather repetitive. You've already got those properties defined in the `BitbucketCommit` and `BitbucketLinks` type definitions, right?

Reply:
Fixed and corrected a duplicate `BitbucketPullRequest` typedefs